persistence/cosmosdb/index.md

```diff
@@ -75,6 +75,24 @@ and registered on the container
 
 snippet: CosmosDBCustomClientProviderRegistration
 
+## Provisioned throughput throttling
```

Comment:
As an alternative, we could also use the term rate limiting instead of throttling. Don't know which is more common, though.